tests/slt/alter_table.slt

```diff
@@ -0,0 +1,19 @@
+statement ok
+create table alter_table(id int primary key, v1 int)
```

Comment:
In fact, the field defaults to Not Null. This probably also explains that when filling the Null value in https://github.com/KipData/KipSQL/pull/104/files#diff-ef2e7111e3dcf8c93251d804dcf4416c7fcc6a99a00aa5fd137a822d90f35eb4, you need to check whether it is allowed to be Null and throw an exception.

Reply:
check in there: https://github.com/KipData/KipSQL/pull/104/files#diff-f2254317cf50142c53882a4e2cbe4715cf85b403c38071ed0edfa7b53a42d986R177

Reply:
> check in there: https://github.com/KipData/KipSQL/pull/104/files#diff-f2254317cf50142c53882a4e2cbe4715cf85b403c38071ed0edfa7b53a42d986R177

It should be necessary to determine whether there is a default value here. When it is not nullable but has a default value, it is probably allowed to execute.